### ISSUE TITLE
Seasons in dataset

### ISSUE THREAD

**Issue by SannaJokela** [GispoCoding/tarmo#58]
Was it so, that the season field was determined by Gispo?

In this case, the season is summer although the POI is a skating rink. Also in skiing routes the season is summer. Actually I did not find any winter-season features. 

![image](https://user-images.githubusercontent.com/23258285/154624541-160617d8-c4dc-4171-bcf8-ba2dc483b6b6.png)

Also nature paths might not be seasonal, so there should be a possibility to have "all seasons" POIs and routes.

**Comment by Rikuoja** [comment]
Yeah currently it is set as "kesä" in the import code https://github.com/GispoCoding/tarmo/blob/main/backend/lambda_functions/lipas_loader/lipas_loader.py#L180

@pavetsu14 knows better if there is any actual season information in Lipas, I have no idea.

**Comment by pavetsu14** [comment]
@SannaJokela @Rikuoja There is no season information in Lipas. This was a field we were supposed to fill in the appropriate value. I will make an issue on the logic we could give the default value for this field.

**Comment by Rikuoja** [comment]
Made a PR on the relevant issue, so this can probably be closed now.
